readme.md

```diff
@@ -1,7 +1,29 @@
 # Count Dracula 🧛🏻
 
-Mock web app to test to test [continous compliance](https://github.com/redbadger/continous-compliance/)
+Mock web app to test [continous compliance](https://github.com/redbadger/continous-compliance/)
 
 ## Infrastructure
 
 To read more about infrastructure, go [here](./infrastructure/readme.md)
+
+## Accessibility
+
+Count Dracula conforms to WCAG-2.1 AA standards. We use [Pa11y](https://pa11y.org/) to automate the accessibility testing. 
+
+Pa11y runs on every push to Github, using a bash script inside web/scripts/a11y.sh. The URLS it tests against are set in web/.pa11yci.json. The json outputs are stored in web/test-results/accessibility-tests.
```

Comment:
Please do you mind emphasises paths like this, e.g: `web/scripts/a11y.sh` or **web/scripts/a11y.sh**